shared/config/happy_config.go

```diff
@@ -168,6 +168,36 @@ func NewHappyConfig(bootstrap *Bootstrap) (*HappyConfig, error) {
 	return config, config.validate()
 }
 
+func NewBlankHappyConfig(bootstrap *Bootstrap) (*HappyConfig, error) {
```

Comment:
`NewHappyConfig()` attempts to locate a `.happy` folder up the folder tree, making it unusable for bootstrapping.